backend/src/test/java/com/ody/mate/service/MateServiceTest.java

```diff
@@ -105,4 +115,69 @@ void findAllByMemberAndMeetingIdException() {
         assertThatThrownBy(() -> mateService.findAllByMemberAndMeetingId(member2, meeting.getId()))
                 .isInstanceOf(OdyBadRequestException.class);
     }
+
+    @DisplayName("Eta상태가 지각 위기인 mate를 콕 찌를 수 있다")
+    @Test
+    void nudgeSuccessWhenLateWarning() {
+        LocalDateTime now = TimeUtil.nowWithTrim();
+        Meeting meeting = new Meeting(
+                "오디",
+                now.toLocalDate(),
+                now.toLocalTime().plusMinutes(1L),
+                Fixture.TARGET_LOCATION,
+                "초대코드"
+        );
+        Meeting oneMinutesLaterMeeting = meetingRepository.save(meeting);
+        Member member1 = memberRepository.save(Fixture.MEMBER1);
+        Mate mate = new Mate(oneMinutesLaterMeeting, member1, new Nickname("콜리"), Fixture.ORIGIN_LOCATION, 10L);
+        mate = mateRepository.save(mate);
+        Eta eta = etaRepository.save(new Eta(mate, 2L));
+
+        mateService.nudge(mate.getId());
+
+        Mockito.verify(getFcmPushSender(), times(1)).sendNudgeMessage(any());
+    }
+
+    @DisplayName("Eta상태가 지각인 mate를 콕 찌를 수 있다")
+    @Test
+    void nudgeSuccessWhenLate() {
+        LocalDateTime now = TimeUtil.nowWithTrim();
+        Meeting meeting = new Meeting(
+                "오디",
+                now.toLocalDate(),
+                now.toLocalTime(),
+                Fixture.TARGET_LOCATION,
+                "초대코드"
+        );
+        Meeting oneMinutesLaterMeeting = meetingRepository.save(meeting);
+        Member member1 = memberRepository.save(Fixture.MEMBER1);
+        Mate mate = new Mate(oneMinutesLaterMeeting, member1, new Nickname("콜리"), Fixture.ORIGIN_LOCATION, 10L);
+        mate = mateRepository.save(mate);
+        Eta eta = etaRepository.save(new Eta(mate, 2L));
+
+        mateService.nudge(mate.getId());
+
+        Mockito.verify(getFcmPushSender(), times(1)).sendNudgeMessage(any());
+    }
+
+    @DisplayName("Eta상태가 지각 위기가 아닌 mate를 찌르면 예외가 발생한다")
```

Comment:
[질문]

지각도 포함되어야 할 것 같아요

```suggestion
    @DisplayName("Eta상태가 지각이나 지각 위기가 아닌 mate를 찌르면 예외가 발생한다")
```

Reply:
반영해주었습니다!